oletools/util/log_helper/_logger_adapter.py

```diff
@@ -0,0 +1,30 @@
+import logging
+from . import _root_logger_wrapper
+
+
+class OletoolsLoggerAdapter(logging.LoggerAdapter):
+    """
+    Adapter class for all loggers returned by the logging module.
+    """
+    _json_enabled = None
+
+    def print_str(self, message):
```

Comment:
This function is meant for things that should always be in the output, such as the banner.
It is present in every logger we created, meaning every oletools' module that has a logger will be able to call it. It is part of the logging module because we need to be able to tell when we are using JSON.

When using JSON we won't print, we will log. If we just printed we would end up with invalid JSON output, something like:
```json
[
msodde 0.53 - http://decalage.info/python/oletools
THIS IS WORK IN PROGRESS - Check updates regularly!
Please report any issue at https://github.com/decalage2/oletools/issues
{ "msg": "some message" }
]
```

We could format the message as JSON and print, but it would lead to an unexpected behavior: say that we want the logger handler to write to a file - some parts of the output would be in the file and others would be in the console.